### PR TITLE
add Gemfile.lock, freeze activesupport.

### PR DESCRIPTION
**Pull request by tknzk**

### CODE REVIEW

File: egov.gemspec [hunk 1]
```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'webmock'
 
   # spec.add_runtime_dependency 'signer'
-  spec.add_runtime_dependency 'activesupport'
+  spec.add_runtime_dependency 'activesupport', '~> 5.2'
```
Comment (meganemura) [inline egov.gemspec]: Would you please relax activesupport version to ">= 5.2", since we use this gem in Rails 6.

Comment (tknzk) [reply egov.gemspec]: :(

Comment (tknzk) [reply egov.gemspec]: fixed: https://github.com/kufu/kiji/pull/15/commits/943eaec4c5f20c8518f18daebbc4539c47acca08

**Comment by tknzk** [comment]
thanks.

**Pull request merged**